core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -51,6 +51,13 @@ oppia.controller('StateHints', [
     };
 
     $scope.changeActiveHintIndex = function(newIndex) {
+      var currentActiveIndex = $scope.activeHintIndex;
+      if (Number.isInteger(currentActiveIndex) && (
+        !stateHintsService.displayed[currentActiveIndex].hintText)) {
```

Comment:
You can indent this by 2 more to distinguish it from the next line (which is the one that actually starts an inner scope).